lib/beaker/dsl/install_utils.rb

```diff
@@ -807,8 +807,8 @@ def install_puppet_from_msi( host, opts )
         end
 
         if host['is_cygwin'].nil? or host['is_cygwin'] == true
-          dest = "/cygdrive/c/Windows/Temp/#{host['dist']}.msi"
-          on host, "curl -O #{dest} #{link}"
+          dest = "#{host['dist']}.msi"
```

Comment:
Wouldn't it be out of scope at that point?

If I'm barking up nonexistent trees, ignore me.